lightstep/resource_inferred_service_rule.go

```diff
@@ -39,7 +38,7 @@ func getResourceInferredServiceRuleSchema() map[string]*schema.Schema {
 			ValidateFunc: validation.StringLenBetween(0, 190),
 		},
 		"description": {
-			Description:  "An optional description to describe the rule and what services it should infer",
+			Description:  "A description of the rule and what services it should infer",
```

Comment:
This was added to the "Optional" section in the docs, so no need to write that it's optional